django_mongodb_backend/base.py

```diff
@@ -192,11 +198,27 @@ def _rollback(self):
     def set_autocommit(self, autocommit, force_begin_transaction_with_broken_autocommit=False):
         self.autocommit = autocommit
 
+    def _close(self):
+        pass
+
```

Comment:
What function calls this?

Reply:
Normally `DatabaseWrapper.close()` but it's also called by some tests.